install_config/web_console_customization.adoc

```diff
@@ -855,6 +855,32 @@ xref:../install_config/configuring_authentication.adoc#OpenID[OpenID] identity
 providers, which require visiting an external URL to destroy single sign-on
 sessions.
 
+[[changing-the-membership-whitelist]]
+== Changing the Membership Whitelist
+
+The default whitelist in the membership page will show a certain subset of Cluster Roles
+(such as 'admin', 'basic-user', 'edit', etc), as well as any custom roles defined within
+a project.
+
+For example, to add your own set of custom Cluster Roles to the whitelist:
+
```

Comment:
s/Cluster Roles/cluster roles

Reply:
Is it should be `add your own set of custom roles to the whitelist`? Since we support both `Cluster Roles` and `Roles`(project/namespace scope)

Reply:
You can add `ClusterRoles` and `Roles`, I suppose, but our filter already will approve all `Roles`, so there is no reason to add them.  It might read a little funny, but I wanted to be specific `cluster roles` to help imply this point.  We assume that a new `Role` is created for a pretty important reason (since there are already so many roles to choose from) so should show by default.